sdk/servicebus/azure-servicebus/tests/async_tests/mgmt_tests/test_mgmt_queues_async.py

```diff
@@ -299,6 +306,99 @@ async def test_async_mgmt_queue_create_with_queue_description(self, servicebus_n
             await mgmt_service.delete_topic(topic_name)
             await mgmt_service.close()
 
+    @CachedResourceGroupPreparer(name_prefix='servicebustest')
+    @CachedServiceBusNamespacePreparer(name_prefix='servicebustest', sku='Premium')
+    async def test_async_mgmt_queue_premium_create_with_queue_description(self, servicebus_namespace_connection_string,
+                                                                  **kwargs):
+        mgmt_service = ServiceBusAdministrationClient.from_connection_string(servicebus_namespace_connection_string)
+        await clear_queues(mgmt_service)
+        queue_name = "dkldf"
+        queue_name_2 = "vjiqjx"
+        queue_name_3 = "rekocd"
+
+        await mgmt_service.create_queue(
+            queue_name,
+            auto_delete_on_idle=datetime.timedelta(minutes=10),
+            dead_lettering_on_message_expiration=True,
+            default_message_time_to_live=datetime.timedelta(minutes=11),
+            duplicate_detection_history_time_window=datetime.timedelta(minutes=12),
+            enable_batched_operations=True,
+            # enable_express=True,
+            # enable_partitioning=True,
+            lock_duration=datetime.timedelta(seconds=13),
+            max_delivery_count=14,
+            max_size_in_megabytes=3072,
+            # requires_duplicate_detection=True,
+            requires_session=True,
+            max_message_size_in_kilobytes=12345
+        )
+
+        await mgmt_service.create_queue(
+            queue_name_2,
+            auto_delete_on_idle="PT10M1S",
+            dead_lettering_on_message_expiration=True,
+            default_message_time_to_live="PT11M2S",
+            duplicate_detection_history_time_window="PT12M3S",
+            enable_batched_operations=True,
+            lock_duration="PT13S",
+            max_delivery_count=14,
+            max_size_in_megabytes=3072,
+            requires_session=True
+        )  # default max_message_size_in_kilobytes is 1024
+
+        with pytest.raises(HttpResponseError):
+            await mgmt_service.create_queue(
+                queue_name_3,
+                max_message_size_in_kilobytes=1023  # min allowed is 1024
+            )
+
+        with pytest.raises(HttpResponseError):
+            await mgmt_service.create_queue(
+                queue_name_3,
+                max_message_size_in_kilobytes=102401  # max allowed is 102400
```

Comment:
where are the max and min specified? is there a default value? I don't believe it's specified in the docstring

Reply:
default value is the same as min which is 1024 kb, max is 102400 kb (100 mb). I will update the doc string

Reply:
doc string updated